json-to-pot.js

```diff
@@ -1,9 +1,49 @@
+// More about the structure of .po files:
+// https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html#PO-Files
+
+/*
+ * white-space
+ #  translator-comments
+ #. extracted-comments
+ #: reference…
+ #, flag…
+ #| msgid previous-untranslated-string
+ msgid untranslated-string
+ msgstr translated-string
+ */
+
 const json = require('./src/locales/en.json')
+const fs = require('fs')
 
 const curlyRegex = new RegExp('{[a-z]*}')
 const containsCurlyWord = (string) => curlyRegex.test(string)
 const checkStringForVars = (string) =>
-  containsCurlyWord(string) ? '(Do not translate words in curly braces)' : ''
+  containsCurlyWord(string) ? '(Do not translate words between ###)' : ''
+
+/**
+ * For GlotPress to display warning when the translators try to
+ * replace placeholders with something else, we need to wrap the
+ * placeholders with `###word###`
+ * @param string
+ * @return {string}
+ */
+const replaceVarsPlaceholders = (string) => {
+  if (!containsCurlyWord(string)) {
+    return string
+  }
+  const variable = /{(?<variable>[a-z]*)}/g
```

Comment:
Some placeholders contain a dash

I spotted this string in the pot file: `###title######creator###{marked-licensed}###license###{view-legal}`

This should work based on what I've seen:
```
const variable = /{(?<variable>[a-z-]*)}/g
```

Reply:
```suggestion
  const variable = /{(?<variable>[a-z-]*)}/g
```

Reply:
There's also `{localeCount}` with an uppercase C, so probably worth just making it case-insensitive too.